src/v/cloud_storage_clients/s3_url_style.h

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
I think stuff like this could go into the `cloud_storage_clients/types.h`.

Reply:
Agreed. Fixed!